www/src/data/sidebars/doc-links.yaml

```
@@ -188,26 +188,26 @@
         - title: Styling Your Site
           link: /docs/styling/
           items:
-            - title: Using Layout Components
-              link: /docs/layout-components/
-            - title: Using CSS Modules
+            - title: Standard styling with global CSS files
+              link: /docs/global-css/
+            - title: Locally-scoped styles with CSS Modules
               link: /docs/css-modules/
-            - title: Using Typography.js
-              link: /docs/typography-js/
-            - title: Using CSS-in-JS Library Emotion
-              link: /docs/emotion/
-            - title: Using CSS-in-JS Library Glamor
-              link: /docs/glamor/
-            - title: Using CSS-in-JS Library Styled Components
-              link: /docs/styled-components/
-            - title: Creating Global Styles
-              link: /docs/creating-global-styles/
-            - title: Writing Component CSS
-              link: /docs/component-css/
-            - title: Using PostCSS
-              link: /docs/post-css/
-            - title: Using TailwindCSS
-              link: /docs/tailwind-css/
+            - title: Enhance your styles with CSS-in-JS
+              link: /docs/css-in-js/
+              items:
+                - title: Emotion
+                  link: /docs/emotion/
+                - title: Styled-Components
+                  link: /docs/styled-components/
+            - title: CSS Frameworks and Libraries
```

Since Typography.js comes first, should we list libraries before frameworks? It would also impact the page title and URL.
```suggestion
            - title: CSS Libraries and Frameworks
```

---

Done in e9785b7